### DISCUSSION TITLE
iMessage immediate logout, Facetime login failure

### DISCUSSION THREAD

**Discussion by zooks97** [osy86/HaC-Mini#594] [Q&A]
**Describe the bug**
I launch `Messages.app` and copy or type in my Apple ID and password. They are accepted, and I get an empty window like this:
![image](https://user-images.githubusercontent.com/8887252/121393652-dac76780-c950-11eb-94b1-db42cad2ff6b.png)
If I quickly type in the name of a contact, they are found.
However, after about a second (and before any conversations or chat histories are loaded), I'm dumped back to the login window.

When I open `FaceTime.app`, my Apple ID is automatically filled in, but after entering my password, I'm given a loading circle and the login fails with no error message.

**Expected behavior**
Login in Messages and FaceTime works.

**Configuration**
- HaC Mini version: 3.0.1
- OSX version: 11.4 (20F71)
- NUC model: NUC8i7HVK
- Boot SSD model and protocol: Samsung SSD 970 PRO 512GB, NVMe
- First DIMM model: Corsair Vengeance DDR4-2400 C16 16GB
- Display port used: HDMI (USB-C not working; screen goes black during Apple logo / progress bar on boot)
- Display resolution: 1440p Dell UltraSharp U2721DE
- Any TB3 devices: No 
- Any USB devices: Leopold FC660M (keyboard); Logitech HD Pro Webcam C920
- Any Wi-Fi/Bluetooth devices: BCM94360CS2 (Broadcom BCM43xx 1.0 (7.77.111.1 AirPortDriverBrcmNIC-1680.8))
  - MX Master 3 Mac

**Modifications**
TB patch (attempted to be) applied; no other modifications.

**Comment by osy** [comment]
Can you try reinstalling the package and choose to regenerate unique identifiers?

**Reply by zooks97** [reply]
I tried running the installer with only regenerating the unique identifiers selected, followed by a reboot. The same issue persists (I tried twice for good measure).

**Reply by osy** [reply]
Can you try a different Apple ID?

**Reply by holygeek00** [reply]
I face up with this issue too.

**Reply by holygeek00** [reply]
I have tried many ways to slove this problem, but it did not work at all.

**Reply by zooks97** [reply]
> Can you try a different Apple ID?

I don't have another Apple ID to try, but I'll make one and give it a go.

@holygeek00, have you tried more than one Apple ID?

**Reply by holygeek00** [reply]
sure，but I failed too

**Reply by holygeek00** [reply]
> > Can you try a different Apple ID?
> 
> I don't have another Apple ID to try, but I'll make one and give it a go.
> 
> @holygeek00, have you tried more than one Apple ID?

I have tried to use another effective  series number , and it finally works. Maybe you could use GenSmbios to generate new series number,MLB and uuid.

**Reply by holygeek00** [reply]
@osy Maybe you could write this in your guide.
![Uploading 497ED965-6607-40B3-939A-1F34616F1C86.jpeg…]()

**Comment by scottzoloft** [comment]
In my experience I've always had to call Apple support to get a newly setup hackintosh working with iMessage and Facetime. When you generate SMBIOS info it's not guaranteed Apple will recognize your system info as valid. You can try re-rolling your unique identifiers or just call Apple.

Call Apple support and tell them you can't sign into iMessage or Facetime. Tell them your spoofed serial and identifiers. If they ask where you got the computer just say from a used computer store or a family member and you've just installed the OS fresh. Tier 1 support won't be able to help you but eventually you should be transferred to a tech who can manually authorize your Apple ID sign in attempts from your spoofed machine. They've done this for two machines for me.

**Reply by Gouster4** [reply]
What can I tell them when they ask why I did spoofed serial and identifiers?

**Reply by scottzoloft** [reply]
> What can I tell them when they ask why I did spoofed serial and identifiers?

Don't tell them you did. Just say it's your mom or dad's old work machine and they gave it to you. Don't say anything like "it doesn't look like a Mac" or "It looks weird". Just tell them the year and model of what you're spoofing and the error code you're getting.

When I had to do this, I spoke with someone from Tier 1 or 2 support, they'd go "hmm that's weird..", and eventually they'd escalate the issue to someone who unblocked my serial and identifier combo. Granted this was 4-5 years ago now.

**Reply by Gouster4** [reply]
Ok. Thank you.
